src/signals/incident/components/form/PlainText/index.test.js

```diff
@@ -118,6 +118,22 @@ describe('Form component <PlainText />', () => {
       expect(element).toHaveStyleRule('background-color', '#E6E6E6')
     })
 
+    it('should render plain text default correctly', () => {
+      const props = getProps({
+        ...metaProps,
+        type: '',
+        value: 'Default',
+      })
+
+      render(withAppContext(<PlainText {...props} />))
+
+      expect(screen.getByTestId('plainText')).toBeInTheDocument()
+      expect(screen.getByText(props.meta.value)).toBeInTheDocument()
```

Comment:
This scenario has already been covered by previous tests

Reply:
oké